docs/building-spark.md

```diff
@@ -265,15 +265,13 @@ Change the major Scala version using (e.g. 2.13):
 
     ./dev/change-scala-version.sh 2.13
 
-For Maven, please enable the profile (e.g. 2.13):
+Eenable the profile (e.g. 2.13):
```

Comment:
@sarutak . nit. A typo: `Eenable` -> `Enable`?

Reply:
Ah, I'll fix it. Thanks.